README.md

```diff
@@ -55,6 +55,64 @@ rng.next();
 rng.next();
 ```
 
+### Usage patterns
```

Comment:
Title of this section could be more "restrictive" because it is only relevant for users who want to persist state across upgrades.

Maybe:
```
### Usage patterns to persist state across upgrades

Reply:
Removed